modules/fundamental/src/advisory/service/mod.rs

```diff
@@ -94,7 +94,10 @@ impl AdvisoryService {
             .filtering_with(
                 search,
                 Columns::from_entity::<advisory::Entity>()
-                    .add_columns(source_document::Entity)
+                    .add_column(
+                        source_document::Column::Ingested,
+                        ColumnType::TimestampWithTimeZone,
+                    )
```

Comment:
I believe this is performance culprit. I lazily added all the `source_document::Entity` columns instead of just the one we need for this query, `source_document::Column::Ingested`